### PR TITLE
Landmark Regions: Updated example pages references to latest docs

### PR DESCRIPTION
**Pull request by jongund**
Updated:
* Related reference links
* SkipTo.js to version 4.0 with ARIA theme option
* Updated resources page
* Added screen shot of Orca with a list of landmarks

[Preview link](https://raw.githack.com/w3c/aria-practices/update-landmarks/examples/landmarks/index.html)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/pull/1882.html" title="Last updated on Sep 7, 2021, 12:56 AM UTC (3388fb9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/1882/ea4ade3...3388fb9.html" title="Last updated on Sep 7, 2021, 12:56 AM UTC (3388fb9)">Diff</a>

### CODE REVIEW

File: examples/landmarks/complementary.html [hunk 1]
```diff
@@ -216,11 +216,13 @@ <h2 id="id3">Related Documents</h2>
...
-            "attachElement": "header"
+            "attachElement": "#skip-to-attach",
+
+            "colorTheme": "aria"
           }
         }
       };
     </script>
-    <script type="text/javascript" src="js/SkipTo.min.js"></script>
+    <script type="text/javascript" src="https://cdn.disability.illinois.edu/skipto.min.js"></script>
```
Comment (jnurthen) [inline examples/landmarks/complementary.html:226]: Should the w3c docs have a dependency on this?

File: examples/landmarks/HTML5.html [hunk 2]
```diff
@@ -158,11 +159,13 @@ <h2 id="id3">Related Documents</h2>
...
-            "attachElement": "header"
+            "attachElement": "#skip-to-attach",
+
+            "colorTheme": "aria"
           }
         }
       };
     </script>
-    <script type="text/javascript" src="js/SkipTo.min.js"></script>
+    <script type="text/javascript" src="https://cdn.disability.illinois.edu/skipto.min.js"></script>
```
Comment (mcking65) [inline examples/landmarks/HTML5.html:169]: We shouldn't have this external dependency

File: examples/landmarks/at.html [hunk 3]
```diff
@@ -243,11 +245,13 @@ <h2 id="id3">Related Documents</h2>
...
-            "attachElement": "header"
+            "attachElement": "#skip-to-attach",
+
+            "colorTheme": "aria"
           }
         }
       };
     </script>
-    <script type="text/javascript" src="js/SkipTo.min.js"></script>
+    <script type="text/javascript" src="https://cdn.disability.illinois.edu/skipto.min.js"></script>
```
Comment (mcking65) [inline examples/landmarks/at.html:255]: Same issue with external dependency on UI CDN

**Review by mcking65 — CHANGES_REQUESTED** [review]
Thank you @jongund for keeping this up to date. We clearly have an issue for the redesign project here. Because we are not using respec on these pages, we have the hard coded hrefs. We need to find a way to get rid of them in the redesign or these hrefs will need constant attention on a lot of pages.

I see an external dependency on UI CDN. Can you please remove that?

**Comment by mcking65** [comment]
@a11ydoer please review the visual aspects of this PR, especially screen shots.

**Review by a11ydoer — APPROVED** [review]
Screen shots of each screen reader look good! Thanks, @jongund!

**Comment by a11ydoer** [comment]
@mcking65 This is ready for merge.

**Comment by jongund** [comment]
@mcking65 
I updated the CDN reference to use a local version.

**Review by mcking65 — APPROVED** [review]

**Pull request merged**